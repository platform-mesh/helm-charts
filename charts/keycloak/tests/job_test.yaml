suite: istio
templates:
  - job-keycloak-configuration.yaml
  - sa.yaml
release:
  namespace: openmfp-system
values:
  - ../test-values.yaml
tests:
  - it: matches the snapshot
    set:
      imagePullSecretOverride: "ghcr-credentials"
      istio:
        quitSidecar: true
    asserts:
      - matchSnapshot: {}