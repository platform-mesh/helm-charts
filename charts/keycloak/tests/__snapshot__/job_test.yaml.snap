matches the snapshot:
  1: |
    apiVersion: v1
    imagePullSecrets:
      - name: ghcr-credentials
    kind: ServiceAccount
    metadata:
      name: keycloak-client-creation
